sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Set-SPOTenant.md

```diff
@@ -4487,6 +4491,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -RestrictResourceAccountAccess
+This feature is currently in preview and may not be available in your tenant.
```

Comment:
Have a sentence describing the property. Then add this preview statement after that. Try this:

```
Start with your param description here.

> [!NOTE]
> This feature is currently in preview and may not be available in your tenant.
```